index.html

```diff
@@ -4409,6 +4409,17 @@
     <div style='position:fixed;margin-left:56%;margin-top:310px;width:100px;height:28px;z-index:2;background-color:rgb(0,130,20);text-align:center;font-size:19px;font-family:trebuchet ms;padding-top:4px;color:white;border-radius:2px' id='replaceAmulet'>Replace</div>
 </div>
 
+<div id='amuletUIWarn' style='position:fixed;left:50%;top:50%;width:250px;height:350px;z-index:2;transform:translate(-50%,-50%) scale(1.25,1.25);background-color:rgb(255,255,100);border-radius:4px;display:none'>

```

Comment:
The plan is to have the amuletUI direct to here when the user clicks "replace medal". Then the code will check the old and the new stats to warn the user when they are accidentally about to downgrade their medal.